src/course-outline/card-header/CardHeader.scss

```diff
@@ -5,7 +5,7 @@
   .item-card-header__title-btn {
     justify-content: flex-start;
     padding: 0;
-    width: fit-content;
+    width: 100%;
```

Comment:
@pkulkark 

Although this solves the issue at hand, it does make the edit button as well as the `Draft (Unpub..)` tag look quite odd and squished

![image](https://github.com/user-attachments/assets/cae02ae0-2508-49bc-8b39-7590a278162f)

I would suggest the following changes which seems to solve our problem without affecting other elements:
```suggestion
    width: fit-content;
    max-width: 62%;
```

Reply:
@kaustavb12 I'm not sure that's going to work. With `width: fit-content` the truncated line gets wrapped which was the original issue. Maybe we can play around with max-width while keeping `width: 100%` but it feels like that's still going to throw off the styling one way or the other.  The oc instance studio seems to be down at the moment. Will test it once it's up and running and update.